### PR TITLE
forgot-reset-password

### PR DESCRIPTION
**Pull request by ShubhamPadiyaTatvasoft**

### CODE REVIEW

File: src/app/core/auth/components/forgot-password/forgot-password.component.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    font-weight: 700;
+    font-size: 1.8rem;
+    line-height: 2.5rem;
+    padding-bottom: 0.4rem;
+    color: #333;
+    margin-bottom: 4rem;
+  }
+
+  .forgot-links-wrapper {
+    display: flex;
```
Comment (Trusha-Tatvasoft) [inline src/app/core/auth/components/forgot-password/forgot-password.component.scss:101]: this class is not used please check all class and remove which are not in use

File: src/app/core/auth/components/reset-password/reset-password.component.scss [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    text-align: center;
+    border-bottom: 1px solid #fcc110;
+    font-weight: 700;
+    font-size: 1.8rem;
+    line-height: 2.5rem;
+    padding-bottom: 0.4rem;
+    color: #333;
+    margin-bottom: 4rem;
+  }
+
```
Comment (Trusha-Tatvasoft) [inline src/app/core/auth/components/reset-password/reset-password.component.scss:113]: same for above

Comment (ShubhamPadiyaTatvasoft) [reply src/app/core/auth/components/reset-password/reset-password.component.scss]: done

File: src/app/core/auth/components/forgot-password/forgot-password.component.scss (hunk 1 above)
Comment (ShubhamPadiyaTatvasoft) [reply src/app/core/auth/components/forgot-password/forgot-password.component.scss]: done

**Review by Trusha-Tatvasoft — APPROVED** [review]

**Review by divyapalTatva — APPROVED** [review]

**Pull request merged**